src/reducers/paths.reducer.js

```diff
@@ -26,11 +26,11 @@ export default (state: State = initialState, action: Action) => {
   switch (action.type) {
     case ADD_PROJECT:
     case IMPORT_EXISTING_PROJECT_FINISH: {
-      const { path, project } = action;
+      const { path: projectPath, project } = action;
```

Comment:
It probably makes sense to rename the `action` itself to have `projectPath` instead of `path`. I'd rather the variable be consistent file-to-file rather than being renamed if we need the `path` module.